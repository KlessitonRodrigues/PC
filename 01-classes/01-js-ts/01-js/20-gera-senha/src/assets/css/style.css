body {
    font-family: sans-serif;
}

main {
    display: flex;
    flex-direction: column;
    align-items: center;
}

label {
    display: flex;
    flex-wrap: wrap;
    align-items: center;
    padding: .5rem 0;
    color: #444;
    font-weight: bold;
    user-select: none;
    font-size: 1.1rem;
}

label:hover {
    color: #74A;
}

input[type="text"], input[type="number"] {
    border: none;
    outline: none;
    border-bottom: 2px solid #74A;
    width: 100%;
    padding: .3rem;
    font-size: 1rem;
    font-weight: bold;
}

input[type="checkbox"] {
    margin-right: .2rem;
}

button {
    min-width: 4rem;
    padding: .3rem;
    font-size: 1.1rem;
    letter-spacing: 1px;
    border-radius: 4px;
    background-color: #DDD;
    border: none;
    color: #000;
    cursor: pointer;
    margin: .5rem 0;
}

button:hover {
    background-color: #DBC;
}

button:active {
    transform: scale(.95, .95);
}

h1 {
    color: #444;
}

.line {
    height: 2px;
    margin-bottom: 1rem;
    background-image: linear-gradient(to right, #d12bfa,#4613ff, #00f7ff);
}

.checkbox-group {
    display: flex;
    justify-content: space-between;
}

.password {
    margin: 2rem 0;
    padding: 1rem;
    font-size: 1.4rem;
    background-color: #EEE;
    border-radius: 5px;
}