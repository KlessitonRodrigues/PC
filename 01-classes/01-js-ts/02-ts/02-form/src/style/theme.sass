$background: #0E0F19
$onBackground: #DDD
$surface: #E5E9EC
$onSurface: #222
$primary: #7B4B94
$onPrimary: #FFF
$secondary: #F29E4C
$onSecondary: #222
$radius: .25rem
