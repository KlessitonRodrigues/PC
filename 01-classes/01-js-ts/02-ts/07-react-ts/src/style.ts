import { createGlobalStyle } from "styled-components";

const GlobalStyle = createGlobalStyle`
  * {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
  }

  body {
    font-family: sans-serif;
    background-color: #222;
    color: #ddd;
  }

  html{
    font-size: 14.5px;
  }

  @media(min-width: 380px){
    html{
      font-size: 15px;
    }
  }

  @media(min-width: 768px){
    html{
      font-size: 15.5px;
    }
  }

  @media(min-width: 1024px){
    html{
      font-size: 16px;
    }
  }
`;

export default GlobalStyle;
