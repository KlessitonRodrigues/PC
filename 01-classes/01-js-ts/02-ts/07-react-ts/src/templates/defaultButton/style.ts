import styled from "styled-components";

export const Container = styled.button`
    background-color: #333;
    color: #FFF;
`