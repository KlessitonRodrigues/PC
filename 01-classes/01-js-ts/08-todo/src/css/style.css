@import "./materialcolors.slim.min.css";
* {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}

body {
    background-color: var(--blue-gray-700);
}

input[type=text] {
    font-size: 1rem;
    display: inline-block;
    border: 1px solid var(--border-black);
    padding: .6rem;
    border-radius: 3px;
    width: 100%;
    box-shadow: .1rem .1rem .2rem var(--shadow);
    color: var(--gray-700);
}

input:hover, input:focus {
    outline: 0;
    border: 1px solid var(--blue-300);
}

button {
    background-color: var(--blue-900);
    color: var(--white);
    text-transform: uppercase;
    letter-spacing: .04rem;
    font-weight: bold;
    margin: .5rem 0;
    padding: .6rem;
    border: 2px solid var(--border-black);
    border-radius: 4px;
    box-shadow: .1rem .1rem .2rem var(--shadow);
}

button:hover {
    background-color: var(--blue-700);
}

.container {
    background-color: var(--white);
    margin-bottom: 1rem;
    padding: 1.2rem .7rem;
}

/*  ADD-TODO  */

.add-todo {
    display: flex;
    flex-flow: column nowrap;
    align-items: flex-start;
}

/*  SHOW-TODO  */

.show-todo {
    display: flex;
    flex-direction: column;
    flex-wrap: wrap;
}

.todo {
    display: flex;
    flex-flow: column nowrap;
    align-items: flex-start;
}

.todo.done {
    background-color: var(--blue-gray-900);
}

.todo.done .todo-text {
    background-color: var(--blue-gray-100);
    border: 2px solid var(--border-black);
}

.todo-text {
    margin-top: .5rem;
    border: 2px solid var(--border-black);
    padding: .6rem;
    width: 100%;
    word-wrap: break-word;
}

/*  MEDIA-QUERIE */

@media(min-width: 40rem) {}