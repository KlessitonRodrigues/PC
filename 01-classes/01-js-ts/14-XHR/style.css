:root {
    --primary: #527;
    --secondary: #ADE;
    --background: #DDD;
    --surface: #224;
    --onBackground: #222;
    --onSurface: #EEE;
    --shadow: 0 1px 3px #444;
}

* {
    margin: 0;
    padding: 0;
}

body {
    background-color: var(--background);
    color: var(--onBackground);
    font-family: sans-serif;
}

li {
    list-style: none;
}

a {
    text-decoration: none;
    color: unset;
}

a:hover {
    filter: invert(.2)
}

nav {
    background-color: var(--primary);
    padding: 1rem;
    font-weight: bold;
    font-size: 1.2rem;
    color: var(--onSurface);
    box-shadow: var(--shadow);
}

nav ul li {
    display: inline-block;
    margin-right: 1rem;
}

.content {
    display: flex;
    justify-content: center;
}

.page {
    width: 100%;
    max-width: 900px;
    margin: 2rem .5rem 0 .5rem;
    padding: 1rem;
    box-shadow: var(--shadow);
    border-radius: 6px;
    background-color: var(--secondary);
}