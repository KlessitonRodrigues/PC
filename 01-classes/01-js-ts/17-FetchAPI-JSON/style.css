body{
    font-family: sans-serif;
}

table{
    width: 100%;
    max-width: 700px;
    background-color: #EEE;
    padding: 1rem;
}

table td{
    padding: .4rem 0;
}