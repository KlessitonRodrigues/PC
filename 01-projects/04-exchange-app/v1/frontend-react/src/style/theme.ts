export const theme = {
  colors: {
    primary: '#1B62CC',
    primaryOpacity: '#1B62CC33',
    primaryVariant: '#1A3763',
    background: '#F3F3F4',
    surface: '#FFFFFF',
    fontColor: '#616161',
    border: '#5554'
  },
  screens: {
    xs: '380px',
    sm: '480px',
    md: '768px',
    lg: '1280px'
  }
}

export type Theme = typeof theme

declare module 'styled-components' {
  export interface DefaultTheme extends Theme {}
}
