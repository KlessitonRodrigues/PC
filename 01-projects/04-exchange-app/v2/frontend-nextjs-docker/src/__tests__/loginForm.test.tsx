import React from 'react'
import renderer from 'react-test-renderer'
import { ThemeProvider } from 'styled-components'

import LoginForm from '../components/loginForm'
import { defaultTheme } from '../style/theme'

describe('LoginForm Component', () => {
  it('Should match with the previous snapshot', () => {
    const component = renderer
      .create(
        <ThemeProvider theme={defaultTheme}>
          <LoginForm />
        </ThemeProvider>
      )
      .toJSON()

    expect(component).toMatchSnapshot()
  })
})
