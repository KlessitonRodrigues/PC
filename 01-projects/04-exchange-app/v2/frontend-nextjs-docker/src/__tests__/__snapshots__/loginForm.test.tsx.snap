// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm Component Should match with the previous snapshot 1`] = `
<div
  className="sc-dkPtRN jngKKC"
>
  <h1
    className="sc-hKwDye gkvLJB"
  />
  <form
    className="sc-eCImPb dtbBTV"
  >
    <span
      className="sc-kDTinF cxNxed"
    >
      <div
        className="sc-gsDKAQ gvEsYY"
        onClick={[Function]}
      >
        <Britain.svg />
        EN
      </div>
    </span>
    <h2
      className="sc-jRQBWg ekNuTk"
    >
      Login
    </h2>
    <span
      className="sc-jrQzAO eRFCWQ"
    >
      
    </span>
    <label
      className="sc-gKclnd eMtMUi"
    >
      <input
        className="sc-bdvvtL hEEjYL"
        onChange={[Function]}
        style={
          Object {
            "color": "#1B62CC",
            "fontSize": "1.5rem",
            "paddingTop": "0.5rem",
          }
        }
        value=""
      />
    </label>
    <label
      className="sc-gKclnd eMtMUi"
    >
      <input
        className="sc-bdvvtL hEEjYL"
        onChange={[Function]}
        style={
          Object {
            "color": "#1B62CC",
            "fontSize": "1.5rem",
            "paddingTop": "0.5rem",
          }
        }
        type="password"
        value=""
      />
    </label>
    <div
      className="sc-furwcr LjklX"
    >
      <button
        className="sc-iCfMLu hvycKJ"
        onClick={[Function]}
        type="button"
      />
      <span
        className="sc-pVTFL bLNBJH"
        onClick={[Function]}
      />
    </div>
  </form>
</div>
`;
